### ISSUE TITLE
Set up GitLink

### ISSUE THREAD

**Issue by asbjornu** [Pomona/Pomona#86]
[GitLink](/GitTools/GitLink) is an alternative to SymbolSource that it would be nice if Pomona supported.

**Comment by BeeWarloc** [comment]
Yes, I think this looks promising. It would probably double the size of the nupkgs, but they're not really that big anyway..
